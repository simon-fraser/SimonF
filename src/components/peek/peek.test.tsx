import { render } from '@testing-library/react';
import { it, describe, expect } from 'vitest';
import React from 'react';
import Peek from './peek'

describe('components/peek', () => {
  it('will render correctly', () => {
    const {asFragment} = render(<Peek />)

    expect(asFragment()).toMatchSnapshot()
  })
})