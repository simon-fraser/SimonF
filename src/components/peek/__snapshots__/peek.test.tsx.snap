// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/peek > will render correctly 1`] = `
<DocumentFragment>
  <aside
    class="absolute bottom-2 right-2 z-20 opacity-50 transition hover:scale-150 hover:opacity-100"
  >
    <a
      href="https://github.com/simon-fraser/SimonF"
      target="_blank"
      title="View the source of this site"
    >
      <svg
        fill="none"
        height="24"
        stroke="white"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="100%"
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="18"
          cy="18"
          r="3"
        />
        <circle
          cx="6"
          cy="6"
          r="3"
        />
        <path
          d="M13 6h3a2 2 0 0 1 2 2v7"
        />
        <line
          x1="6"
          x2="6"
          y1="9"
          y2="21"
        />
      </svg>
    </a>
  </aside>
</DocumentFragment>
`;
